consensus/vbft/service.go

```diff
@@ -180,7 +182,12 @@ func (self *Server) Receive(context actor.Context) {
 		self.handleBlockPersistCompleted(msg.Block)
 	case *p2pmsg.ConsensusPayload:
 		self.NewConsensusPayload(msg)
-
+	case *p2pComm.EmergencyGovCmd:
+		if msg.Cmd == p2pComm.EmgGovStart {
+			self.StartEmergency(msg.Height)
```

Comment:
I think we could just stop consensus / start consensus, rather than handling these emergency functions in vbft?